nvflare/private/fed/server/job_cmds.py

```diff
@@ -45,13 +45,13 @@ def get_spec(self):
                     usage="list_jobs [-n name_prefix] [-s study_prefix] [-d] [job_id_prefix]",
                     handler_func=self.list_jobs,
                 ),
-                CommandSpec(
-                    name="delete_job",
-                    description="delete a job",
-                    usage="delete_job job_id",
-                    handler_func=self.delete_job,
-                    authz_func=self.authorize_job,
-                ),
+                # CommandSpec(
+                #     name="delete_job",
+                #     description="delete a job",
+                #     usage="delete_job job_id",
+                #     handler_func=self.delete_job,
+                #     authz_func=self.authorize_job,
+                # ),
```

Comment:
We can also remove this along with delete_job function since they are in previous commits if we ever want to bring them back.

Reply:
We are planning on addressing it 2.1.1, so this will be modified soon regardless